dali/test/python/test_torch_pipeline_rnnt.py

```diff
@@ -377,10 +377,6 @@ def rnnt_train_pipe(
     audio_shape = audio.shape(dtype=types.INT32)
     orig_audio_len = audio_shape[0]
 
-    # If we couldn't move to GPU earlier, do it now
-    if device == "gpu" and frame_splicing_subsample > 1:
-        audio = audio.gpu()
-
     if pad_amount > 0:
         audio_len = orig_audio_len + 2 * pad_amount
         padded_audio = dali_reflect_pad_graph(audio, pad_amount)
```

Comment:
In the problematic case those values weren't actually used - the could could have been reorganized to work with the old executor, but why not use all the goodies?